### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Djon-Latii**
Please see my changes. 
Added paths and changed the project structure

### CODE REVIEW

**Review by Nicelx — APPROVED** [review]

File: src/pages/contactList/ContactList.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    componentDidMount() {
+        this.loadContacts();
+
+    }
+
+    render() {
+        const { contacts } = this.state;
+        return (
+            <div className="contact-list" style={{width: '100%',
```
Comment (AlexBychkov) [inline src/pages/contactList/ContactList.js]: let's move this style to a separate file like ContactList.css

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
-          <Route path='/component' component={Home} />
-         {/*  <Redirect from='/' to='/home'/> */}
-        </Switch>
-      </div>
-    );
-  }
+  return (
+    <div className="App">
+        <Route path="/">
+          {true ? <Home /> : <Redirect from='/' to="/login" />}
```
Comment (AlexBychkov) [inline src/App.js:17]: hm, I think there we should have all the components and routes, for example ContactList

**Review by AlexBychkov — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 2 above)
Comment (Djon-Latii) [reply src/App.js:17]: I transferred all components to Home. This componet see only registуred user

Comment (Djon-Latii) [reply src/App.js:17]: image

**Pull request merged**